BioMed.Api/BioMed.Domain/ResourceParameters/DepartmentResourceParameters.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace BioMed.Domain.ResourceParameters
+{
+    public class DepartmentResourceParameters : ResourceParametersBase
+    {
+        public override string OrderBy { get; set; } = "name";
+
+        protected override int MaxPageSize { get; set; } = 50;
```

Comment:
Why did you override **MaxPageSize** property, if all properties have the same value,
wouldn't it be better if you just change in repository base?